phpunit.xml

```diff
@@ -4,4 +4,12 @@
     <testsuite name="Net">
         <directory>src/Tests</directory>
     </testsuite>
-</phpunit>
\ No newline at end of file
+     <filter>
```

Comment:
I'm not familiar with this feature. What exactly is this doing?

Reply:
This can define the code coverage folder to tell `PHPUnit` where can do that.